### DISCUSSION TITLE
I want to know how to show the train acc and loss during training

### DISCUSSION THREAD

**Discussion by leesangjoon1** [open-mmlab/mmclassification#1213] [Q&A]
I train a model. And validate it per 10 intervals.
But it shows only validation loss
I want to know how to show the train acc and loss during training
I added evaluation = dict(interval=10, metric='accuracy', save_best='loss', rule=greater')
but it doesn't work
please help me
Thank you :)

**Comment by mzr1996** [comment]
MMClassification always prints the loss during training, I think it's because the log interval is too large to show your training loss.
If you are using mmcls master branch, please modify the log interval in your config file:
```python
# It's usually in the `default_runtime.py`
log_config = dict(
    interval=100,  # Modify here to a lower value, like 1.
    ...)
```
If you are using mmcls new 1.x branch, please modify the log interval in your config file:
```python
# It's usually in the `default_runtime.py`
default_hooks = dict(
    ...
    logger=dict(type='LoggerHook', interval=100),  # Modify here to a lower value, like 1.
    ...
)
```

Also, if you want to print the accuracy of the training data batch, please add `cal_acc=True` to the head config, like
```python
model = dict(
    ...
    head = dict(
        ...,
        cal_acc=True,
    ),
)
